test/events/failEvent.json

```diff
@@ -0,0 +1,23 @@
+{
+    "event": "issues", 
+    "payload": {
+      "action": "opened",
+      "issue": {
+        "body": "Fix your broken thigns!",
+        "title": "ur thing is broken fix it!!",
+        "user": {
+          "login": "hiimbex"
+        },
+        "labels": []
```

Comment:
So these are just fake payloads that I used for testing; however, I double checked and it's not actually necessary for the mocking, so I'll remove them 👍 But I believe I copied them and that the API does return an empty list if there are no labels, but I'm not positive and would have to double check